doc/projects.rst

```diff
@@ -48,19 +47,12 @@ This pane allows you to perform all the same operations as a normal Spyder :doc:
 Working with version control
 ============================
 
-Spyder has basic integration with the `Git`_ distributed version control system.
-You can commit or browse (in the ``gitk`` GUI) a file, directory or the entire repository via the commands in the context menu for the relevant object (right-click).
+The :guilabel:`Project` pane, like :doc:`fileexplorer`, has basic integration with the `Git`_ distributed version control system.
+You can commit or browse a file, directory or the entire repository via the commands in the context menu.
```

Comment:
Perhaps we should add a screenshot or gif below this to display the functionality we're talking about? It seems missing to me.

Reply:
We discussed this at some length; I originally described this functionality in more detail here since there is so little to talk about with regard to the Projects pane and Git repos have some relationship to projects, but I forgot about it when we were doing the Files pane docs and we ended up talking about it there instead, where we already have a GIF that shows exactly this. While ideally we would have had that here instead, now that pane is already done I see little value in duplicating it again here, other than a brief mention of this support.

Reply:
@ccordoba12 Any final thoughts on this?

Reply:
I suggest to add a link to the [File operations](http://docs.spyder-ide.org/current/fileexplorer.html#file-operations) section of `Files`, also telling people to look at the end of it to see how git operations work.

Or for a simpler way to reference this, we could create a new section in `Files` called `Git operations` and link directly to it from here.

Reply:
Okay, I'll have to do that as a commit. Standby.

Reply:
Because I have to rebase it anyway as it wasn't branched from the latest master, and thus the `Files` pane doc needs a rebase anyway, I'll just do the rebase along with the change as part of the rebase, and if it looks good we can be ready to merge.

Reply:
Done.

Reply:
Great! As just took a look at your changes in Netlify and they're exactly what i expected. Thanks!